internal/templates/data/common/py/boot.py

```diff
@@ -4,6 +4,6 @@
 if __name__ == "__main__":
     logging.basicConfig(
         level=logging.INFO,
-        format='[%(levelname)s] %(message)s',
+        format='[%(levelname).5s] %(message)s',
```

Comment:
can i make this pad right with whitespace if the levelname is less than 5 so that the spacing is consistent?